app/build.gradle

```diff
@@ -15,10 +15,55 @@ ext.versionPatch = 0
 def productRootDir = "./"
 def productBinDir = new File(productRootDir, "bin")
 def productLibAndroidDir = new File(productRootDir, "libs")
-def archUsed = "armeabi-v7a,x86"
+def archUsed = "armeabi-v7a,arm64-v8a,x86,x86_64"
 def usedNdfList = ["Irises.ndf"]
 
 android {
+
+    packagingOptions {
+        exclude 'member-search-index.zip'
+        exclude 'jquery-ui.overrides.css'
+        exclude 'jquery/jquery-ui.min.css'
+        exclude 'index-all.html'
+        exclude 'jquery/jszip/dist/jszip.min.js'
```

Comment:
This is a very big file @hkaluuma  . For that case, i would seek review from @druchniewicz